### PR TITLE
[Azure.Monitor.Query] Update autorest.md

### LINKED ISSUE #24623: Add x-ms-enum.modelAsString=false to single value enums in Azure.Monitor.Query
`seal-single-value-enum-by-default` in `autorest.md` is a temporary flag. When it stops working, enums with single value will no longer be treated as constants. To avoid this, add `x-ms-enum.modelAsString=false` to all enums that should remain constants, e.g.: e.g.:
https://github.com/Azure/autorest.csharp/blob/25d9b624ae1e6c5faf65fbb8176e718f85c6b4c8/test/TestProjects/BodyAndPath-LowLevel/BodyAndPath-LowLevel.json#L106 (in this example, however, parameter is still created cause EnumName2 has 2 values).

### PR DESCRIPTION
**Pull request by nisha-bhatia**
Fixes https://github.com/Azure/azure-sdk-for-net/issues/24623
Removed seal-single-value-enum-by-default: true from autorest.md as it is not needed and there are no codegen changes.

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by nisha-bhatia** [comment]
https://github.com/Azure/azure-sdk-for-net/issues/24623

File: sdk/monitor/Azure.Monitor.Query/src/autorest.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ input-file:
 generation1-convenience-client: true
 modelerfour:
     lenient-model-deduplication: true
-    seal-single-value-enum-by-default: true
```
Comment (JoshLove-msft) [inline sdk/monitor/Azure.Monitor.Query/src/autorest.md:15]: Can you mention in the PR description that this was not necessary and that there were no changes after regenerating?

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**